packages/pipeline/src/utils/isElementOverflowed.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Helper function to check whether a DOM element is overflowed.
```

Comment:
What do you mean by overflowed? It extends beyond the viewport? `isElementOverflownViewport`? Not sure I like that any better.

Also FWIW I can easily see a feature where this function and some other functions get upstreamed to the lib package for use with other components. We'll have to keep our eye out for when we need to start using them elsewhere.  Very exciting addition!

Reply:
Yeah naming things is hard. What I mean by overflowed, is really the past tense of `overflow` – which in a CSS context means the content is clipped when an element's it is too big to fit in a specified area.

I'm not a fan of `Overflown`, specifically the flown part, as I think that would be closer to the past tense of "fly".

🤷‍♂ Not sure what else to suggest.

Reply:
Yeah let's keep the name, but let's inline this comment description as it's gold jerry, gold.

Reply:
Resolved by b383cf5